base_transaction_id/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2012 Yannick Vaucher, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{'name': 'Base transaction id for financial institutes',
+ 'version': '11.0.1.0.0',
+ 'author': "Camptocamp,Odoo Community Association (OCA)",
+ 'maintainer': 'Camptocamp',
+ 'category': 'Hidden/Dependency',
+ 'depends': [
+     'sale',
+ ],
+ 'website': 'https://www.odoo-community.org/',
```

Comment:
https://github.com/OCA/account-reconcile

Reply:
@yvaucher remember to update this